examples/offline-wallet-signing-message.js

```diff
@@ -0,0 +1,22 @@
+const { Message } = require('@dashevo/dashcore-lib');
+const { Wallet } = require('../src/index');
+
+const mnemonic = 'never citizen worry shrimp used wild color snack undo armed scout chief';
+const walletOpts = {
+  offlineMode: true,
+  network: 'testnet',
+  mnemonic,
+};
+const wallet = new Wallet(walletOpts);
+const account = wallet.getAccount();
+
+const startSigningMessage = () => {
+  const message = new Message('Hello world!');
+
+  const idPrivateKey = account.getIdentityHDKey().privateKey;
+
+  const signed = account.sign(message, idPrivateKey);
+  const verify = message.verify(idPrivateKey.toAddress().toString(), signed.toString()); // true
```

Comment:
Is the comment on this line supposed to be removed?

Reply:
Not really, I mean, it's a way to know what the input looks like without having to run it. 
It's probably bad, may be it is sufficient to already have that info in the doc for message.verify in dashcore-lib ?

Reply:
I see. Probably fine either way.